web/src/SidebarAccount.tsx

```diff
@@ -252,7 +252,7 @@ function SidebarAccount(props: SidebarAccountProps) {
       <MenuContentSignInLink>
         Or{" "}
         <strong>
-          <a href={props.tiltCloudSchemeHost}>Sign In</a>
+          <a href={props.tiltCloudSchemeHost} target="_blank">Sign In</a>
```

Comment:
at the risk of derailing this PR, i don't understand what this link does? It says "Sign In" but it's just a link to Tilt Cloud.

@hyu is it possible that this is just vestigial, and you meant to delete it when you added the real sign-in flow above?